app/services/user_signup.rb

```diff
@@ -0,0 +1,47 @@
+class UserSignup
+  attr_reader :error_message
+  
+  def initialize(user)
+    @user = user
+  end
+  
+  def sign_up(stripe_token, invitation_token)
```

Comment:
Not all sign_ups will be with `invitation_token` it would be cool if you used a default `nil` so you can use sign_up without needing to pass nil .